From a0478203982889d3d95ff650a54f6b4fbea3d744 Mon Sep 17 00:00:00 2001
From: Amit Kumar bhaumik <amitx.bhaumik@intel.com>
Date: Tue, 17 May 2022 12:41:21 +0000
Subject: [PATCH] Redefining CONFIG_BOOTCOMMAND for secure boot

Redefining CONFIG_BOOTCOMMAND for secure boot feature specific to ctl
platform

Tested: Build the image and tested in the ctl and egs platform and its booted
fine.

Signed-off-by: Amit Kumar bhaumik <amitx.bhaumik@intel.com>
Upstream-Status: Pending
---
 include/configs/aspeed-common.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/configs/aspeed-common.h b/include/configs/aspeed-common.h
index dafa867d82..d3ba37e1ba 100644
--- a/include/configs/aspeed-common.h
+++ b/include/configs/aspeed-common.h
@@ -68,9 +68,6 @@
 #define CONFIG_ENV_SIZE			0x10000
 #endif
 
-#undef CONFIG_BOOTCOMMAND
-#define CONFIG_BOOTCOMMAND		"bootm 20b00000"
-
 #undef CONFIG_ENV_OFFSET
 #define CONFIG_ENV_OFFSET		0xa0000
 
-- 
2.17.1

